build/aca-machine-init.sh

```diff
@@ -117,17 +117,12 @@ echo "7--- installing pulsar dependacies ---" && \
 echo "8--- building alcor-control-agent"
 cd $BUILD/.. && cmake . && make
 
-echo "9--- rebuilding br-tun and br-int"
-ovs-ctl --system-id=random --delete-bridges restart
-ovs-vsctl add-br br-int -- add-br br-tun
-ovs-vsctl \
-    -- add-port br-int patch-tun \
-    -- set interface patch-tun type=patch options:peer=patch-int \
-    -- add-port br-tun patch-int \
-    -- set interface patch-int type=patch options:peer=patch-tun
-
-ovs-ofctl add-flow br-tun "table=0, priority=1,in_port="patch-int" actions=resubmit(,2)"
-ovs-ofctl add-flow br-tun "table=2, priority=0 actions=resubmit(,22)"
+if [ "$1" == "delete-bridges" ]; then
```

Comment:
@cj-chung please take a look to confirm this approach is okay for our environment.